tests/v1/test_distributions.py

```diff
@@ -0,0 +1,43 @@
+from itertools import product
+
+import numpy as np
+import pytest
+from scipy.integrate import cumulative_trapezoid
+from scipy.stats import kstest
+
+from petab.v1.distributions import *
+from petab.v2.C import *
+
+
+@pytest.mark.parametrize(
+    "distribution, transform",
+    list(
+        product(
+            [
+                Normal(1, 1),
+                LogNormal(2, 1),
+                Uniform(2, 4),
+                Laplace(1, 2),
+                LogLaplace(1, 0.5),
+                ParameterScaleNormal(1, 1),
+                ParameterScaleLaplace(1, 2),
+                ParameterScaleUniform(1, 2),
+            ],
+            [LIN, LOG, LOG10],
+        )
+    ),
+)
+def test_sample_matches_pdf(distribution, transform):
+    """Test that the sample matches the PDF."""
+    np.random.seed(1)
+    N_SAMPLES = 10000
+    distribution.transform = transform
+    sample = distribution.sample(N_SAMPLES)
+
+    # pdf -> cdf
+    def cdf(x):
+        return cumulative_trapezoid(distribution.pdf(x), x)
+
+    # Kolmogorov-Smirnov test to check if the sample is drawn from the CDF
+    _, p = kstest(sample, cdf)
+    assert p > 0.05, (p, distribution)
```

Comment:
Should this be higher than `0.05`? e.g. `p > 0.5` should also always work for 10000 samples, just guessing. Anyway, fine as is.

Reply:
With p > 0.5 this will fail frequently. Even with p > 0.05 it fails often enough so that I set a seed.

Reply:
That's surprising! I would have guessed that 10,000 samples from a "simple" distribution should be enough to say "these distributions look similar"... anyway fine :+1: